src/gui_common/TranslationSiteButton.cs

```diff
@@ -0,0 +1,14 @@
+using Godot;
+using System;
+
+/// <summary>
+///   Shows a version label
+/// </summary>
+public class TranslationSiteButton : Button
+{
+    public override void _Process(float delta)
```

Comment:
Don't use _Process to get button inputs. The button class has the pressed signal for this